### DISCUSSION TITLE
Spring4Shell zero-day (CVE-2022-22965)

### DISCUSSION THREAD

**Discussion by LucasOlivera-andreani** [provectus/kafka-ui#1788] [Q&A] [answered]
Dear, 
I wanted to know if you are aware of Spring4Shell zero-day 
https://www.computerweekly.com/news/252515360/Spring4Shell-zero-day-sprung-on-security-teams
and if this affects the kafka-ui tool in any way.

Thanks for your help

**Comment by Haarolean** [comment, marked as answer]
Hi, thanks for bringing attention onto this issue.

Our app is not affected in any build type we provide (either jar file or docker image).
